backend/api/schedule/mutations.py

```diff
@@ -42,8 +45,70 @@ class UpdateOrCreateSlotItemResult:
     updated_slots: typing.List[ScheduleSlot]
 
 
+@strawberry.type
+class BookScheduleItemError:
+    message: str
+
+
+@strawberry.type
+class ReleaseScheduleItemBookingError:
+    message: str
+
+
+@strawberry.type
+class ScheduleItemBookingStatus:
+    schedule_item: ScheduleItemType
+
+
 @strawberry.type
 class ScheduleMutations:
+    @strawberry.mutation(permission_classes=[IsAuthenticated])
+    def release_schedule_item_booking(
+        self, info, id: strawberry.ID, conference: str
+    ) -> typing.Union[ScheduleItemBookingStatus, ReleaseScheduleItemBookingError]:
+        user = info.context["request"].user
+
+        schedule_item_booking = ScheduleItemBooking.objects.filter(
+            user=user, schedule_item_id=id, schedule_item__conference__code=conference
+        ).first()
+
+        if not schedule_item_booking:
+            return ReleaseScheduleItemBookingError(message=_("No booking found"))
```

Comment:
this could a BookingNotFound error ;)

Reply:
same for the other ones, I imagine the FE using the typename information to show an appropriate message 😊